### PR TITLE
Added new theme

### PR DESCRIPTION
**Pull request by victorstein**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -5,7 +5,22 @@ module.exports = {
...
+        primary: {
+          dark: '#d73636',
+          DEFAULT: '#d73636'
+        },
+        background: {
+          dark: '#23343b',
+          DEFAULT: '#ffffff'
+        },
+        text: {
+          dark: '#000000',
```
Comment (GustavoGomez092) [inline tailwind.config.js]: Dark Text should be #212121

File: tailwind.config.js [hunk 2]
```diff
@@ -5,7 +5,22 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        primary: '#FF6363'
+        primary: {
+          dark: '#d73636',
+          DEFAULT: '#d73636'
```
Comment (GustavoGomez092) [inline tailwind.config.js]: Color should be #D83636

**Review by GustavoGomez092 — CHANGES_REQUESTED** [review]
Left comments on right colors

File: tailwind.config.js (hunk 1 above)
Comment (victorstein) [reply tailwind.config.js]: Text dark is FFF. DARK is used for dark theme. default will be 212121

File: tailwind.config.js (hunk 2 above)
Comment (victorstein) [reply tailwind.config.js]: updated

**Review by GustavoGomez092 — APPROVED** [review]

**Pull request merged**